.buildkite/scripts/run_models.sh

```diff
@@ -18,4 +18,5 @@ cd integration_tests
 dbt deps
 dbt seed --target "$db" --full-refresh
 dbt run --target "$db" --full-refresh
-dbt test --target "$db"
\ No newline at end of file
+dbt test --target "$db"
```

Comment:
I actually just realized here that we are not testing the incremental strategy in our integration tests 😱.

Would you be able to add another run and test (not full refresh) after the full refresh to effectively test our incremental strategy.